hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -529,19 +529,10 @@ private void checkIterationMoveResults() {
       allFuturesResult.get(config.getMoveTimeout().toMillis(),
           TimeUnit.MILLISECONDS);
     } catch (InterruptedException e) {
+      LOG.warn("Container balancer is interrupted");
       Thread.currentThread().interrupt();
     } catch (TimeoutException e) {
-      long timeoutCounts = moveSelectionToFutureMap.entrySet().stream()
-          .filter(entry -> !entry.getValue().isDone())
-          .peek(entry -> {
-            LOG.warn("Container move canceled for container {} from source {}" +
-                    " to target {} due to timeout.",
-                entry.getKey().getContainerID(),
-                containerToSourceMap.get(entry.getKey().getContainerID())
-                    .getUuidString(),
-                entry.getKey().getTargetNode().getUuidString());
-            entry.getValue().cancel(true);
```

Comment:
Can I ask why this line is removed in the new function?

Reply:
The cancel part should be needed to handle timeout tasks.

Reply:
@symious This is updated based on review comment: https://github.com/apache/ozone/pull/3751#discussion_r978370000

Reply:
I remember I added this cancelation because of some issue in our cluster: even after the timeout, the request is also handled by datanodes, then many containers have 4 replicas, causing ReplicationManager to clean the redundant one.

The cancelation here can help eliminate these cases.

Reply:
reverted changes and raised PR